Vector.c

```
@@ -93,91 +100,24 @@ void Vector_prune(Vector* this) {
-         j--;
-      }
-      array[j + 1] = t;
-   }
-}
-
-void Vector_quickSortCustomCompare(Vector* this, Object_Compare compare) {
-   assert(compare);
-   assert(Vector_isConsistent(this));
-   quickSort(this->array, 0, this->items - 1, compare);
+ATTR_NONNULL_N(1)
+void Vector_sort(Vector* this, Object_Compare compare, void* context) {
```

Cppcheck (2.20.0)

[style] The function 'Vector_sort' is never used.

(unusedFunction)